app/templates/ui/app/search/search-results.html

```diff
@@ -1,6 +1,6 @@
 <div ng-repeat="result in results">
   <h4>
-    <a ui-sref="root.view({uri: result.uri})">{{ result.uri }}</a>
+    <a ui-sref="root.view({uri: result.uri})">{{ result.title || result.uri }}</a>
```

Comment:
I like having the possibility of showing a title or label instead of uri. It is very common to want to not show the uri. I would not mind at all keeping this..